integration/transaction.go

```diff
@@ -114,9 +115,12 @@ func (m txManager) addAndWait(args ...txn.Arg) ([]byte, error) {
 
 		err = m.n.GetPool().Add(sentTxn)
 		if err != nil {
-			return nil, xerrors.Errorf("failed to Add: %v", err)
+			fmt.Printf("Failed to add transaction: %v", err)
+			continue
```

Comment:
that's indeed odd that an retry-loop failed on the first error, I guess it makes your sense with your change

Reply:
Even the more so that one of the errors is the famous "wrong nonce" error, which happens quite often and needs a new transaction to be created...